### PR TITLE
sdk-py: add versioning for assistants

### PR DESCRIPTION
**Pull request by isahers1**
- Lists all the versions for a given assistant
- Allows you to change assistant to point to specific version

### CODE REVIEW

File: libs/sdk-py/langgraph_sdk/client.py [hunk 1]
```diff
@@ -563,6 +568,50 @@ async def search(
...
+
+            assistant_versions = await client.assistants.get_versions(
+                assistant_id="my_assistant_id"
+            )
+
+        """  # noqa: E501
+
+        return await self.http.get(f"/assistants/{assistant_id}/versions")
+
+    async def change_version(self, assistant_id: str, version: int) -> Assistant:
```
Comment (nfcampos) [inline libs/sdk-py/langgraph_sdk/client.py:590]: maybe should be called set_latest or something?

Comment (isahers1) [reply libs/sdk-py/langgraph_sdk/client.py:590]: added to my todo, will make sure to change before merging

Comment (vbarda) [reply libs/sdk-py/langgraph_sdk/client.py:590]: might make sense to keep these consistent in the API as well. maybe `update_version` / `set_version`?

**Review by dqbd — APPROVED** [review]

File: libs/sdk-js/src/client.ts [hunk 2]
```diff
@@ -313,6 +318,47 @@ export class AssistantsClient extends BaseClient {
...
+  }
+
+  /**
+   * Change the version of an assistant.
+   *
+   * @param assistantId ID of the assistant.
+   * @param version The version to change to.
+   * @returns The updated assistant.
+   */
+  async setVersion(assistantId: string, version: number): Promise<Assistant> {
```
Comment (dqbd) [inline libs/sdk-js/src/client.ts:356]: Should be `setLatest` to mirror Python

**Pull request merged**